Gruntfile.js

```diff
@@ -654,7 +654,8 @@ module.exports = function( grunt ) {
 		"test:functional",
 		"uglify",
 		"compare_size",
-		"commitplease"
+		"commitplease",
+        	"update-authors"
```

Comment:
This is now a mix of spaces and tabs. Replace the 8 spaces at the start with 2 tabs, 3 tabs indent in total.